src/app.js

```diff
@@ -37,5 +37,30 @@ app.get('/online-users', async (req, res) => {
   }
 })
 
+app.post('/create-game', async (req, res) => {
+  try {
+    const { users } = req.body
```

Comment:
What happens if we send a request without a `users` property? What happens if the `users` property is not an array? What happens if `users` doesn't have 2 entries? What happens if those 2 entries are empty string?